core/src/test/scala/org/apache/spark/UnpersistSuite.scala

```diff
@@ -31,7 +31,7 @@ class UnpersistSuite extends SparkFunSuite with LocalSparkContext with TimeLimit
     rdd.count
     assert(sc.persistentRdds.isEmpty === false)
```

Comment:
You're welcome to grep for `\.isEmpty ={2,3} false` and make them .`nonEmpty` just while we're here, but this looks good.

Reply:
Only found this one using `grep`， thanks